project/dagger-query-ui.sh

```diff
@@ -12,5 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+xdg-open src/com/google/daggerqueryui/index.html
+start src/com/google/daggerqueryui/index.html
 open src/com/google/daggerqueryui/index.html
```

Comment:
I think you want to conditionally execute `xdg-open` or `open` depending on the platform, otherwise either one of the commands will always fail and confuse the user.

You can do this with an if-else in bash, see https://stackoverflow.com/a/8597411

Alternatively just make available two different scripts, say:
* dagger-query-ui-linux.sh
* dagger-query-ui-macos.sh

Reply:
Ok, using if-else in bash looks pretty straightforward, I [have added two checks](https://github.com/googleinterns/dagger-query/pull/71/commits/e4a939e9e592a5b8c450ed72aab61370b139ddf4): one for OSX, one for Linux. Thank you 🚀